src/frontend/components/CartPageCard/CartPageCard.css

```diff
@@ -0,0 +1,68 @@
+.cart__item {
+    background-color: var(--white-color);
+    box-shadow: #32325d40 0px 6px 12px -2px, #0000004d 0px 3px 7px -3px;
+    gap: 3rem;
+    padding: 0 3rem;
+}
+
+.cart__img {
+    height: 30vh;
+    flex-basis: 30%;
+}
+
+.cart__copy {
+    align-items: flex-start;
+    justify-content: space-around;
+    padding: 2rem;
+    flex-basis: 50%;
+}
+
+.cart__copy-btns {
+    align-items: center;
+    gap: 1rem;
+}
+
+.btn-transparent--primary {
+    background-color: transparent;
+    border: none;
+    color: var(--primary-color);
+}
+
+.btn-transparent--primary:hover {
+    background-color: var(--hover-color);
+}
+
+.cart__action {
+    align-items: center;
+    justify-content: space-around;
+    flex-basis: 20%;
+}
+
+.cart__product--quantity__container {
+    align-items: center;
+    gap: 5px;
+}
+
+.cart__product--quantity-btn {
+    border-radius: 50%;
+    width: 3rem;
+    height: 3rem;
+    border: none;
+    background-color: var(--primary-color);
+    color: var(--white-color);
+    vertical-align: middle;
+    box-shadow: #00000040 0px 54px 55px, #0000001f 0px -12px 30px, #0000001f 0px 4px 6px, #0000002b 0px 12px 13px, #00000017 0px -3px 5px;
+    cursor: pointer;
+}
+
+.cart__product--quantity-btn:hover {
+    background-color: var(--secondary-color);
+}
+
+.cart__product--quantity {
+    font-size: 1.6rem;
+    background-color: var(--hover-color);
+    width: 3rem;
+    text-align: center;
+    padding: 5px;
```

Comment:
use only rem or px .

Reply:
it was recommended by vikram to use px under 10 px